### PR TITLE
Packet lock update after rebase

### PR DESCRIPTION
**Pull request by anthonySchafrik**

### CODE REVIEW

File: client/src/components/EnterPurchase.jsx [hunk 1]
```diff
@@ -29,32 +60,34 @@ class EnterPurchase extends React.Component {
     } = this.props;
 
     return (
-      <Wrapper>
-        <h1>Purchase Amount</h1>
+      <Wrapper data-test="enter-purchase">
+        <h1>Enter potential purchase</h1>
```
Comment (jpdaprato) [inline client/src/components/EnterPurchase.jsx]:
```suggestion
        <h1>Purchase Amount</h1>
```

File: client/src/Home/Home.jsx [hunk 2]
```diff
@@ -18,7 +18,7 @@ class Home extends Component {
   render() {
     const { isAuthenticated } = this.props.auth;
     return (
-      <div className="container">
+      <Wrapper className="container">
```
Comment (jpdaprato) [inline client/src/Home/Home.jsx]:
```suggestion
      <div className="container">
```

File: client/src/Home/Home.jsx [hunk 3]
```diff
@@ -36,7 +36,7 @@ class Home extends Component {
             to continue.
           </h4>
         )}
-      </div>
+      </Wrapper>
```
Comment (jpdaprato) [inline client/src/Home/Home.jsx]:
```suggestion
      </div>
```

File: client/src/components/InputAmount.jsx [hunk 4]
```diff
@@ -45,7 +45,7 @@ class InputAmount extends React.Component {
     const { isEditting } = this.state;
     if (isEditting === false) {
       return (
-        <label onClick={this.changeLabelToInput}>
+        <label style={{ color: "red" }} onClick={this.changeLabelToInput}>
```
Comment (deduced) [inline client/src/components/InputAmount.jsx]: You can make this a styled component using `styled('label')`

File: client/src/components/EnterPurchase.jsx [hunk 5]
```diff
@@ -29,32 +60,34 @@ class EnterPurchase extends React.Component {
...
         <InputAmount
+          data-test="input-amount"
           handlePurchaseInput={handlePurchaseInput}
           purchaseAmount={purchaseAmount}
         />
-        <div>
-          <h2>Cash Flow</h2>
+        <CashFlowMeterDiv>
+          <h2 style={{ marginRight: "30px" }}>Cash Flow</h2>
```
Comment (jpdaprato) [inline client/src/components/EnterPurchase.jsx]: Pull out and put in a styled block above.

File: client/src/components/GoblinAdvice.jsx [hunk 6]
```diff
@@ -1,6 +1,6 @@
 import React from "react";
 import { Link } from "react-router-dom";
-import goblin from "../images/goblin.png";
+// import goblin from "../images/goblin.png";
```
Comment (jpdaprato) [inline client/src/components/GoblinAdvice.jsx]:
```suggestion
```

File: client/src/components/InputAmount.jsx (hunk 4 above)
Comment (jpdaprato) [inline client/src/components/InputAmount.jsx]: Remove and put in a styled block.

File: client/src/components/TopSpendingGraphs.jsx [hunk 7]
```diff
@@ -64,12 +64,19 @@ const TopSpendingGraphs = ({ recurring, categories, shop }) => {
   };
 
   return (
-    <div style={{ height: "50%", width: "50%" }}>
+    <div
+      style={{ height: "50%", width: "50%" }}
```
Comment (jpdaprato) [inline client/src/components/TopSpendingGraphs.jsx]: Remove and move

**Pull request closed without merging**